eunkyung/controllers/likeController.js

```diff
@@ -0,0 +1,11 @@
+const likeService = require('../services/likeService')
+
+
+const likePost = async (req, res) => {
```

Comment:
함수명을 지을 때 행위와 대상 자체에 대해 명시해주는 것도 가독성을 높이는 방법 중 하나입니다.
예를 들어 게시글에 유저의 id 값을 받아 좋아요 데이터를 추가한다. 라는 기능을 구현할 때는 다음과 같이 작성할 수 있습니다.
```suggestion
const createLike = async (req, res) => {
```

Reply:
In [eunkyung/controllers/likeController.js](https://github.com/wecode-bootcamp-korea/42-backend-westagram-1/pull/11#discussion_r1091544801):

> @@ -0,0 +1,11 @@

+const likeService = require('../services/likeService')

+

+

+const likePost = async (req, res) => {

함수명을 지을 때 행위와 대상 자체에 대해 명시해주는 것도 가독성을 높이는 방법 중 하나입니다.
예를 들어 게시글에 유저의 id 값을 받아 좋아요 데이터를 추가한다. 라는 기능을 구현할 때는 다음과 같이 작성할 수 있습니다.

⬇️ Suggested change
-const likePost = async (req, res) => {

+const createLikePostByUserId = async (req, res) => {

In [eunkyung/controllers/postController.js](https://github.com/wecode-bootcamp-korea/42-backend-westagram-1/pull/11#discussion_r1091545266):

> @@ -0,0 +1,35 @@

+const postService = require('../services/postService')

+

+

+const posting = async (req, res) => {

함수명 수정하면 좋을 것 같습니다.
게시글을 생성한다

⬇️ Suggested change
-const posting = async (req, res) => {

+const createPosts = async (req, res) => {

Reply:
In [eunkyung/controllers/likeController.js](https://github.com/wecode-bootcamp-korea/42-backend-westagram-1/pull/11#discussion_r1091544801):

> @@ -0,0 +1,11 @@

+const likeService = require('../services/likeService')

+

+

+const likePost = async (req, res) => {

함수명을 지을 때 행위와 대상 자체에 대해 명시해주는 것도 가독성을 높이는 방법 중 하나입니다.
예를 들어 게시글에 유저의 id 값을 받아 좋아요 데이터를 추가한다. 라는 기능을 구현할 때는 다음과 같이 작성할 수 있습니다.

⬇️ Suggested change
-const likePost = async (req, res) => {

+const createLikePostByUserId = async (req, res) => {

Reply:
In [eunkyung/services/likeService.js](https://github.com/wecode-bootcamp-korea/42-backend-westagram-1/pull/11#discussion_r1091548661):

> @@ -0,0 +1,7 @@

+const likeDao = require('../models/likeDao')

+

+const likePost = async (userId, postId) => {

해당 부분도 함수명을 수정해주세요!
예를 들어 createLike 와 같이 특정 행위가 포함된 함수명이 좀 더 명확한 의미를 전달할 수 있을 것 같습니다.